packages/create-codes/src/index.ts

```diff
@@ -248,15 +248,46 @@ async function run() {
     packageObjs = deepMergeObjects(packageObjs, fse.readJsonSync(packages));
   }
 
-  // TODO: add some example codes
   if (needsE2eTesting) {
+    // Copy example test cases
+    const e2eTestingDir = path.resolve(TEMP_DIR, "__tests__");
```

Comment:
[Q] Is it alright to keep the directory name as `__tests__` in the output project?

Reply:
Sounds good! I'll leave it as it is for now.